queries.js

```diff
@@ -24,3 +24,22 @@ Restaurants
 9 db.restaurants.find({name:'Morris Park Bake Shop'}) //para buscarlo
   db.restaurants.updateOne({'address.street':'Morris Park Ave'}, {$set : {'address.street':'Calle false 123'}}) //el cambio
 10 db.restaurants.deleteMany({'address.zipcode' : '10466'})
+
+Companies
+
+1 db.companies.find({name:'Facebook'})
+2 db.companies.find({category_code:'web'}, {name:1, '_id':0})
+3 db.companies.find({name:'Twitter'}, {name:1, '_id':0, category_code:1, founded_year:1})
+4 db.companies.aggregate( [ {$match: {'category_code' : 'web'}}, {$limit:50} ])
```

Comment:
Curiosa elección de acción para hacer la consulta, para una petición tan simple yo me decantaría por un tradicional find, parece que está más optimizado y su tiempo de respuesta es mejor.
db.companies.find({category_code: 'web'}).limit(50)
Fetched 20 documents in 4300ms 

db.companies.aggregate([{$match: {category_code: 'web'}}, {$limit: 50}])
Fetched 20 record(s) in 5378ms

Os dejo la [docu](https://docs.mongodb.com/manual/core/aggregation-pipeline/) de agregation, que explica mejor cuando se debe usar.